.agents/skills/crafting-effective-readmes/references/standard-readme-spec.md

```
@@ -0,0 +1,242 @@
+
+**Suggestions:**
+- Use http://shields.io or a similar service to create and host the images.
+- Add the [Standard Readme badge](https://github.com/RichardLitt/standard-readme#badge).
+
+### Short Description
+**Status:** Required.
+
+**Requirements:**
+- Must not have its own title.
+- Must be less than 120 characters.
+- Must not start with `> `
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)